src/components/__tests__/KeyControls.js

```diff
@@ -0,0 +1,63 @@
+jest.dontMock('../KeyControls');
+jest.dontMock('../Navigator');
+jest.dontMock('marked');
+
+import React           from 'react';
+import ReactDOM        from 'react-dom';
+import TestUtils       from 'react-addons-test-utils';
+
+const createNavigator = require('../Navigator').default;
+const KeyControls     = require('../KeyControls').default;
+
+describe('KeyControls', () => {
+  let controls, node, navigator, directions, mappings;
+
+  directions = [
```

Comment:
Also this ones I think are reused in the `UIControls` tests. They should live in a fixture